NewEvaluation.css

```diff
@@ -54,8 +57,29 @@ body{
 }
 
 
+.input-candidate{
+	width:33%;
+	border: 0;
+	padding: 0;
+	vertical-align: middle;
+	height: 2em;
+	/*box-shadow: 0px 1px 0px #888888; */
+}
+.input-candidate::placeholder{
+		border-bottom: 1px solid lightgrey;
+		columns: lightgrey;
```

Comment:
If we're styling against the `::placeholder`, that will only be visible when the default placeholder text is visible. Are you sure you didn't want to target `.input-candidate` directly, instead?